siriuspy/siriuspy/sofb/csdev.py

```diff
@@ -63,7 +63,7 @@ class ConstTLines(_csdev.Const):
     DEF_MAX_ORB_DISTORTION = 50  # [um]
     ACQRATE_TRIGMODE = 2  # [Hz]
     ACQRATE_SLOWORB = 60  # [Hz]
-    BPMsFreq = 25.14  # [Hz]
+    BPMsFreq = 10.0  # [Hz]
```

Comment:
Calculating BPMsFreq based on the current RF frequency, I get: 10.476782 Hz. I'm only mentioning it because the old value had more decimal places.

Reply:
done!